### PR TITLE
add telemetry source

### PR DESCRIPTION
**Pull request by MrLeea-13155bc**
**What this PR does / why we need it**:
add Telemetry Srouce 
**Will this PR make the community happier**? 
yes
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**How is this PR tested**
- [ ] unit test
- [ ] e2e test
- [x] other (please specify) 
 no need to test
**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: pkg/k8s/crd/telemetry/utils/utils.go [hunk 1]
```diff
@@ -15,13 +15,13 @@ import (
...
-	URL_DEFAULT_PUBLIC_IP   = "0.0.0.0"
-	TASK_RUN_DEMO_KIND      = "run_shifu_release"
-	SOURCE_SHIFU_CONTROLLER = "shifu_controller"
-	HTTP_CONTENT_TYPE_JSON  = "application/json"
+	URL_EXTERNAL_IP        = "http://cip.cc"
+	URL_IP_LINE            = "<pre>IP"
+	URL_SHIFU_TELEMETRY    = "https://telemetry.shifu.run/shifu-telemetry/"
+	URL_DEFAULT_PUBLIC_IP  = "0.0.0.0"
+	TASK_RUN_DEMO_KIND     = "run_shifu_release"
+	DEFAULT_SOURCE         = "shifu_product"
```
Comment (tomqin93) [inline pkg/k8s/crd/telemetry/utils/utils.go:23]: Let's use `default` as the default source for ***Shifu***

**Review by tomqin93 — COMMENTED** [review]
@MrLeea-13155bc added 2 comments

**Review by tomqin93 — APPROVED** [review]
LGTM

**Comment by tomqin93** [comment]
Will merge this in before release to prevent breaks

**Comment by BtXin** [comment]
Don't have any comments on the PR, but just wondering what are other available telemetry sources, and what does default stands for.

**Comment by MrLeea-13155bc** [comment]
> 



> Don't have any comments on the PR, but just wondering what are other available telemetry sources, and what does default stands for.

shifu demo

File: pkg/k8s/crd/config/edgedevice/bases/shifu.edgenesis.io_telemetryservices.yaml [hunk 2]
```diff
@@ -42,7 +42,7 @@ spec:
                   type: string
                 type: object
               serviceSettings:
-                description: ProtocolSettings defines protocol settings when connecting
+                description: ServiceSettings defines protocol settings when connecting
```
Comment (kris21he) [inline pkg/k8s/crd/config/edgedevice/bases/shifu.edgenesis.io_telemetryservices.yaml:45]: do we need to change the description?  "ServiceSettings defines protocol settings when connecting to an Edge Device"

""ServiceSettings define the service settings when connecting to an Edge Device"

**Comment by BtXin** [comment]
> > 
> 
> > Don't have any comments on the PR, but just wondering what are other available telemetry sources, and what does default stands for.
> 
> shifu demo

Is shifu demo other available source or is shifu demo default?

**Comment by MrLeea-13155bc** [comment]
> > > 
> > 
> > 
> > > Don't have any comments on the PR, but just wondering what are other available telemetry sources, and what does default stands for.
> > 
> > 
> > shifu demo
> 
> Is shifu demo other available source or is shifu demo default?

`default` is just stand for default (install with shifu_install.yaml directly), it will changed to `shifu_demo` on shifu_demo( installed with shifu_demo script)

File: pkg/k8s/crd/config/edgedevice/bases/shifu.edgenesis.io_telemetryservices.yaml (hunk 2 above)
Comment (MrLeea-13155bc) [reply pkg/k8s/crd/config/edgedevice/bases/shifu.edgenesis.io_telemetryservices.yaml:45]: yep,actrually the describe is a mistake

**Review by tomqin93 — CHANGES_REQUESTED** [review]
@MrLeea-13155bc please sync with main

**Review by tomqin93 — APPROVED** [review]

**Pull request merged**